www/js/control/general-settings.js

```diff
@@ -14,8 +16,7 @@ angular.module('emission.main.control',['emission.services',
                                         'emission.plugin.kvstore',
                                         'emission.survey.enketo.demographics',
                                         'emission.plugin.logger',
-                                        'emission.config.dynamic',
-                                        'monospaced.qrcode'])
```

Comment:
Yes we do. I want to use a platform-agnostic method of generating QR codes; and since unified tokens QRs are now being generated on the website join page, I think it would be good to be consistent and use the same method in the app. So what is the join page using?

With QR codes you can encode the same data many ways, and using different generators we would get different images for the same token. That's kind of confusing, so that's why I think it would be good to stick to one implementation.

Reply:
The join page is using http://jeromeetienne.github.io/jquery-qrcode/
Not sure that is super consistent with your goals to remove jquery usage 😦 
The join page was originally intended to be fairly simple, but has gotten a bit more complicated with the QR code generation. I guess we could switch it to react as well, although that is definitely not a high priority.

Reply:
TODO

Reply:
I agree that it is not a big issue, and as I said, if the join webapp code gets more complex, we can just rewrite it in react as well